README.md

```diff
@@ -8,6 +8,7 @@ The following environment variables have to be defined to run the tests:
 
 ```
 $ AUTH_URL=<auth url>/oauth/token
```

Comment:
This should also be changed to:
```suggestion
$ AUTH_URL=<auth url>
```

right?

Reply:
missed that one

Reply:
done